src/Compilers/CSharp/Test/Semantic/Semantics/OutVarTests.cs

```diff
@@ -1066,6 +1068,33 @@ private static void VerifyModelForOutVarInNotExecutableCode(SemanticModel model,
             Assert.Null(model.GetDeclaredSymbol(decl));
         }
 
+        private static void AssertTypeFromOperation(SemanticModel model, TypeSymbol expectedType, DeclarationExpressionSyntax decl)
+        {
+            var variableDeclarator = decl.Ancestors().OfType<VariableDeclaratorSyntax>().FirstOrDefault();
+            if (variableDeclarator?.ArgumentList?.FullSpan.Contains(decl.Span) == true)
+            {
+                // node in argument list is error case which operation doesn't support
```

Comment:
>// node in argument list is error case which operation doesn't support [](start = 16, length = 70)

Please add a link to the issue tracking this bug.

Reply:
opened. https://github.com/dotnet/roslyn/issues/22175